steps/pytorch/validation.py

```diff
@@ -93,3 +93,8 @@ def torch_acc_score_multi_output(outputs, targets, take_first=None):
         accuracies.append(accuracy)
     avg_accuracy = sum(accuracies) / len(accuracies)
     return avg_accuracy
+
+
+def multiclass_segmentation_loss(output, target):
```

Comment:
@apyskir @jakubczakon I think we should discuss where we put metrics and losses. I met such functions in multiple places

Reply:
@kamil-kaczmarek @apyskir Agreed. I think modules objectives.py and metrics.py (copying keras) would make it cleaner. However I don't think this PR should be affected by it